packages/stark-ui/src/modules/table/components/table.component.spec.ts

```diff
@@ -49,6 +49,20 @@ import createSpy = jasmine.createSpy;
 			[rowClassNameFn]="rowClassNameFn"
 			(rowClicked)="rowClickHandler($event)"
 		>
+			<ng-template let-context>
```

Comment:
This example is incorrect, it should be the same as the one you provide in the showcase.
for instance:
```
<ng-container *starkTableRowContent="let rowData = rowData; let cellRawValue = rawValue; let cellDisplayedValue = displayedValue">
    <div *ngIf="cellRawValue > 23; else defaultContent">
        custom content: {{ cellDisplayedValue }}
    </div>
    <ng-template #defaultContent>
        <span>{{ cellDisplayedValue }}</span>
    </ng-template>
</ng-container>
```

Reply:
It seems you forgot to adapt this